### ISSUE TITLE
Pydantic conversion

### ISSUE THREAD

**Issue by Natureboy100** [MrSentinel137/crewai-gemini-pro-vision#1]
Dear Mr.Sentinel.
i have been facing problem running this code and my own code with gemini vision pro or even gemini, as it gives the following pydantic conversion error, is the code running fine by your side? i have run this on colab and locally with everything installed.

here is the error: Unexpected message with type <class 'langchain_core.messages.system.SystemMessage'> at the position 1.

**Comment by RishabhMathur06** [comment]
Yes, same error occurs for me as well. My code shows "exchange" attribute is unknown and the agent ia not able to see the image.
